IOTATransferRestRequestTemplate.py

```diff
@@ -127,19 +124,25 @@ def create_transaction_dictionary(depth=100, request_tag=None):
 
 
 def requestData():
+    # type: () -> Optional[Text]
     request_id = register_request().bytes
     transaction = create_transaction_dictionary(request_tag=request_id)
-    iota.send_transfer(
+
+    # :todo: Populate these values and/or make API object globally-accessible.
+    api = create_iota_object(uri=uri, auth_token=auth_token, seed=seed)
+    api.send_transfer(
         depth=transaction['depth'],
 
         # One or more :py:class:`ProposedTransaction` objects to add to the
         # bundle.
-        transfers=[transaction],
+        transfers=[transaction['transaction-object']],
```

Comment:
`transaction` is a transaction dictionary; `transaction['transaction-object']` is the `ProposedTransaction` object.